pkg/kubectl/cmd/util/helpers.go

```diff
@@ -31,10 +31,11 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/latest"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/client"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+	jsonpatch "github.com/evanphx/json-patch"
```

Comment:
My vim plugin was trying to be smart and removing the import. Used a dumber editor to resolve.